src/utils/constants.tsx

```diff
@@ -68,4 +68,40 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
     href: null,
     color: "#315CF5",
   },
+  FORTMATIC: {
+    connector: fortmatic,
+    name: 'Fortmatic',
+    iconName: 'fortmaticIcon.png',
+    description: 'Login using Fortmatic hosted wallet',
+    href: null,
+    color: '#6748FF',
+    mobile: true
+  },
+  TORUS: {
+    connector: torus,
+    name: 'Torus',
+    iconName: 'torus.jpg',
+    description: 'Login using Torus hosted wallet',
+    href: null,
+    color: '#6748FF',
+    mobile: true
+  }, 
+  AUTHEREUM: {
+    connector: authereum,
+    name: 'Authereum',
+    iconName: 'authereum.png',
+    description: 'Login using Authereum hosted wallet',
+    href: null,
+    color: '#6748FF',
+    mobile: true
+  }, 
+  // BITSKI: {
```

Comment:
same

Reply:
Up to you. I figured we might circle back to this pretty quickly. I can remove it

Reply:
i think we should remove for now - maybe just put a TODO with relevant info to re-implement when the time comes

Reply:
fasho